target/linux/ramips/rt305x/base-files/etc/board.d/02_network

```diff
@@ -206,6 +206,7 @@ ramips_setup_macs()
 	nixcore,x1-8m|\
 	olimex,rt5350f-olinuxino|\
 	olimex,rt5350f-olinuxino-evb|\
+	olimex,rt5350f-olinuxino-evb-plus|\
```

Comment:
Can you provide an overview of MAC addresses, best added to the commit message:
https://openwrt.org/docs/guide-developer/mac.address

Does the device have a label with a MAC address on it?

Reply:
It does not store fixed MAC addresses in flash, it just uses random one.

Reply:
Not at all?

Reply:
AFAIK, nope. Its factory partition only contains calibration parameters.